### DISCUSSION TITLE
Newbie - questions

### DISCUSSION THREAD

**Discussion by ghost** [brackets-cont/brackets#166] [Q&A] [answered]
I've just downloaded Brackets and have some questions:

When I'm writing code, how do I select the auto-completion? For example, if I start to type "font-w" Brackets suggests "font-weight". In other code editors I click the 'tab' button to use the suggestion, but in Brackets this just adds in a tab space. I seem to have to select the suggestion with my mouse.

Will Brackets check/validate/lint my HTML & CSS code?

How do I get rid / close the 'Getting Started' folder in the left side bar?

**Comment by duto-nm** [comment]
@Mark-eee, let me try to help you:

1. To use the suggestion of the auto-completion you must hit 'enter' instead of 'tab';

2. Brackets usually shows in red when you insert something that can broke your code. For example: if you duplicate a closing tag, the next tag will be highlighted with red color;

3. I'm not sure, but if I remember well the Getting Started folder will always appear in the left side bar if you don't have any other folder loaded into Brackets.

**Reply by ghost** [reply]
Thank you!

2. Are there 'linter' extensions I can add to check my site/files?
3. Am I correct in thinking that there is no way to not have a folder in the left side bar? I think I'm going to find that frustrating. I might make a blank folder and open that, rather than have the Getting Started folder. Unless there are benefits to having a Folder in Brackets?

**Reply by duto-nm** [reply]
2. Yes, there is some extensions which can do that, but you need to see if they are not deprecated. You can find them in the Extension Manager (File > Extension Manager), then search for 'Lint';
3. Exactly, there is no way to not have a folder in it. It is good if you are working on a big project with many files, it makes easier to access them. Although, you can still edit files there are not in the folder of the side bar.

**Comment by kirtisingh3008** [comment, marked as answer]
Bracket gives all the auto suggestions in a dropdown-content where we can pick one which we want have a look at the picture attached below, so no point of tab and Getting started will not be there once you open a folder on which you want to work on.
![Screenshot (632)](https://user-images.githubusercontent.com/70806481/158402022-777c8219-2866-44a7-8313-8652496cc727.png)

If you still face any problems, please let us know, Thank You!
